ensembldl.sh

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env bash
```

Comment:
This file has execute permissions. Unfortunately, GitHub does not show this, so I'm commenting on the shebang. Please `chmod -x ensembldl.sh`.

Reply:
done